### PR TITLE
LPD-25817 - playwright - Add a test that asserts that preselected value is checked in the MultiSelect

### PR DESCRIPTION
**Pull request by kresimir-coko**

### CODE REVIEW

**Comment by kresimir-coko** [comment]
ci:test:frontend

**Comment by kresimir-coko** [comment]
ci:report:22056066

**Comment by kresimir-coko** [comment]
No unique failures reported, I will push a commit that removes relevant Poshi test/s

File: modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-admin/selectionFilters.spec.ts [hunk 1]
```diff
@@ -122,4 +123,88 @@ test.describe('Filters in Data Set Manager', () => {
...
 		});
 	});
+
+	test('Preselected filter values are checked in the multiSelect', async ({
+		filtersPage,
+		page,
+		picklistApiHelpers,
+	}) => {
+		await test.step('Navigate to the Filters tab', async () => {
+			await filtersPage.goto({
```
Comment (dsanz) [inline modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-admin/selectionFilters.spec.ts:133]: We can avoid the step as it's [part of `beforeEach()` ](https://github.com/liferay-frontend/liferay-portal/pull/4267/commits/c6703434da053c60bf5739ec8a17152142f2e683#diff-e86a8fda98500d2aea9059c1cf05bdc5d2428c4d19067b5f7f148d016defa609R59-R63)

File: modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-admin/selectionFilters.spec.ts [hunk 2]
```diff
@@ -122,4 +123,88 @@ test.describe('Filters in Data Set Manager', () => {
...
+			const picklist = await picklistApiHelpers.getPicklist(picklistName);
+
+			const preselectedValuesMultiSelect = page.locator(
+				'.form-control.form-control-tag-group.input-group'
+			);
+
+			await expect(preselectedValuesMultiSelect).toBeVisible();
+
+			await expect(preselectedValuesMultiSelect).toContainText(
+				picklist.listTypeEntries[0].name
```
Comment (dsanz) [inline modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-admin/selectionFilters.spec.ts:190]: nice!

**Comment by dsanz** [comment]
LGTM ;) just a minor nit to remove a redundant step of the test

**Comment by kresimir-coko** [comment]
@dsanz I've done my investigation regarding https://liferay.atlassian.net/browse/LPD-26410 and extending the current test to cover it, too.

The conclusion is that that situation is already covered in this line:
```js
await expect(preselectedValuesMultiSelect).toContainText(
	picklist.listTypeEntries[0].name // evaluates to "Sample Value"
);
```
image

**Comment by kresimir-coko** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase [hunk 3]
```diff
@@ -461,88 +461,6 @@ definition {
...
-				radioLabel = "Multiple");
-		}
-
-		task ("Then the Selection and Preselected Values option are displayed") {
-			AssertElementPresent(
-				key_fieldName = "Select a default value for your filter",
-				locator1 = "DataSet#PRESELECT_VALUE_INPUT");
-		}
-	}
-
```
Comment (dsanz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase:647]: :heart:

**Comment by markocikos** [comment]
PW parts will conflict with https://github.com/brianchandotcom/liferay-portal/pull/151698, we'll see who gets to do the fixes :slightly_smiling_face:

**Pull request closed without merging**